src/core/constraint_template.jl

```diff
@@ -81,6 +83,22 @@ function constraint_ohms_yt_from(pm::GenericPowerModel, branch)
     return constraint_ohms_yt_from(pm, f_bus, t_bus, f_idx, t_idx, g, b, c, tr, ti, tm)
 end
 
+### Branch - Loss Constraints DC LINES###
+
+""
+function constraint_ohms_yt_dc(pm::GenericPowerModel, dcline)
+    i = dcline["index"]
+    f_bus = dcline["f_bus"]
+    t_bus = dcline["t_bus"]
+    f_idx = (i, f_bus, t_bus)
+    t_idx = (i, t_bus, f_bus)
+    br_status = dcline["br_status"]
+    loss0 = dcline["loss0"]
```

Comment:
Note that if `pm.ref` is setup correctly, then all components with status 0 are filtered out.  So you do not need to consider the status at this point.